tar/bsdtar.c

```diff
@@ -80,6 +80,7 @@ __FBSDID("$FreeBSD: src/usr.bin/tar/bsdtar.c,v 1.93 2008/11/08 04:43:24 kientzle
 #include "bsdtar.h"
 #include "crypto.h"
 #include "dirutil.h"
+#include "elasticarray.h"
```

Comment:
Is this necessary?  I don't think we use any elasticarray functions directly in here?